server/index.js

```diff
@@ -35,6 +37,9 @@ server.connection({
   }
 });
 
+// TODO clean this up, but at least the method is exposed
+server.method('queue.createVisit', createVisit);
```

Comment:
I didn't want to `require` the queue more than once, seemed like foul things might occur. Not sure if this is the right way to instantiate it once, though - a plugin seems better. @dannycoates thoughts on the best way to attach a singleton thing (like a work queue) to a hapijs server?